src/contracts/interfaces/IEigenPod.sol

```diff
@@ -130,7 +133,7 @@ interface IEigenPod {
      */
     function verifyBalanceUpdate(
```

Comment:
`verifyBalanceUpdates`

Reply:
?

Reply:
I'm reading @gpsanant 's comment as a suggestion on renaming.
am assuming this function can now take in a batch? documentation also needs updating for this function.

Reply:
no it doesn't take a batch, which is why I don't love the new name

Reply:
oh OK. if no batch input then existing name seems correct. resolving this thread.